modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/parser/java/utils/MinMuleVersionUtils.java

```diff
@@ -535,6 +560,18 @@ private static ResolvedMinMuleVersion calculateParameterTypeMinMuleVersion(Type
     return parameterMMV;
   }
 
+  private static boolean isLinkageError(Throwable t) {
+    t = t.getCause();
```

Comment:
not considering that t itself could be a linkeage errror

Reply:
Done.